lib/openstudio/workflow/util/measure.rb

```diff
@@ -474,6 +474,17 @@ def apply_measure(registry, step, options = {}, energyplus_output_requests = fal
 
               begin
                 if energyplus_output_requests
+
+                  # Just in case this workflow-gem is used (via bundle) in an
+                  # older OS SDK version, where the base class doesn't have it,
+                  # we check for existence to begin with
+                  if measure_object.respond_to?(:modelOutputRequests)
+                    # If there is an overidden method in the measure itself: Warn
+                    if measure_object.method(:modelOutputRequests).owner == measure_object.class
+                      logger.error "Reporting Measure at '#{measure_path}' has a 'modelOutputRequests' method, but that is only available in the C++ CLI, it will NOT be run."
+                    end
+                  end
```

Comment:
Before running the energyPlusOutputRequests, check if there is an overriden one and just log an error saying it's not going to be run.